app/src/main/java/com/ivyclub/contact/ui/onboard/contact/AddContactFragment.kt

```diff
@@ -0,0 +1,76 @@
+package com.ivyclub.contact.ui.onboard.contact
+
+import android.annotation.SuppressLint
+import android.os.Bundle
+import android.provider.ContactsContract
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.view.animation.AnimationUtils
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.fragment.app.viewModels
+import com.ivyclub.contact.R
+import com.ivyclub.contact.databinding.FragmentAddContactBinding
+import com.ivyclub.contact.util.BaseFragment
+import com.ivyclub.data.model.PhoneContactData
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class AddContactFragment : BaseFragment<FragmentAddContactBinding>(R.layout.fragment_add_contact){
+
+    private lateinit var contactAdapter: ContactAdapter
+    private lateinit var contactList: MutableList<PhoneContactData>
+    private val viewModel: AddContactViewModel by viewModels()
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        contactList = mutableListOf()
+        initRecyclerView()
+        initButtons()
+    }
+
+    private fun initRecyclerView() {
+        contactAdapter = ContactAdapter()
+        binding.rvContactList.adapter = contactAdapter
+    }
+
+    private fun initButtons() {
+        binding.button.setOnClickListener {
+            val buttonAnimation = AnimationUtils.loadAnimation(requireContext(),R.anim.button_down)
+            val textAnimation = AnimationUtils.loadAnimation(requireContext(),R.anim.text_gone)
+            val recyclerViewAnimation = AnimationUtils.loadAnimation(requireContext(),R.anim.recyclerview_fade_in)
+            binding.button.startAnimation(buttonAnimation)
+            binding.textView.startAnimation(textAnimation)
+            binding.rvContactList.visibility = View.VISIBLE
+            binding.rvContactList.startAnimation(recyclerViewAnimation)
+            binding.button.isClickable = false
+            binding.button.text = "시작하기"
+            contactAdapter.submitList(getContact())
+        }
+        binding.btnCommit.setOnClickListener {
+            viewModel.savePeople(contactAdapter.addList)
+            requireActivity().finish()
+        }
+    }
```

Comment:
```suggestion
    private fun initButtons() = with(binding) {
        button.setOnClickListener {
            val buttonAnimation = AnimationUtils.loadAnimation(requireContext(),R.anim.button_down)
            val textAnimation = AnimationUtils.loadAnimation(requireContext(),R.anim.text_gone)
            val recyclerViewAnimation = AnimationUtils.loadAnimation(requireContext(),R.anim.recyclerview_fade_in)
            button.startAnimation(buttonAnimation)
            textView.startAnimation(textAnimation)
            rvContactList.visibility = View.VISIBLE
            rvContactList.startAnimation(recyclerViewAnimation)
            button.isClickable = false
            button.text = "시작하기"
            contactAdapter.submitList(getContact())
        }
        btnCommit.setOnClickListener {
            viewModel.savePeople(contactAdapter.addList)
            requireActivity().finish()
        }
    }
```
이미 아시겠지만 이렇게 해주시면 조금 더 깔끔할 것 같습니다..